packages/cli/src/lib/cmds/shared.ts

```diff
@@ -84,3 +88,58 @@ export async function updateVersions(
     appVersionName: appVersionName,
   };
 }
+
+export async function generateManifestChecksumFile(manifestFile: string,
+    prompt: Prompt): Promise<void> {
+  fs.readFile(manifestFile, async function(err, data) {
+    if (err) {
+      prompt.printMessage(err.toString());
+      return;
+    }
+    const csFile = path.join(process.cwd(), 'manifest-checksum.txt');
+    const sum = crypto.createHash('sha1').update(data).digest('hex');
+    await fs.promises.writeFile(csFile, sum);
+  });
+}
+
+export async function updateProject(
+    skipVersionUpgrade: boolean,
+    appVersionName: string,
+    prompt: Prompt = new InquirerPrompt(),
+    directory: string,
+    manifest: string): Promise<boolean> {
+  const targetDirectory = directory || process.cwd();
+  const manifestFile = manifest || path.join(process.cwd(), 'twa-manifest.json');
+  const twaManifest = await TwaManifest.fromFile(manifestFile);
+  twaManifest.generatorApp = APP_NAME;
+
+  const features = twaManifest.features;
+  // Check that if Play Billing is enabled, enableNotifications must also be true
```

Comment:
I know you didn't introduce these comments, but do you mind cleaning them up a bit? (Adding full stops, changing "billing" to "Play Billing" in the one below)

Reply:
done.